### ISSUE TITLE
Back-Matter/Resource only allows rlink or base64, not both

### ISSUE THREAD

**Issue by brianrufgsa** [usnistgov/OSCAL#533]
# Describe the bug

In prior discussions, there was consensus that the resource assembly in back-matter would allow both rlink and base64 elements. Tools would then have the option of deciding which to prefer, asking the user, or even validating the two against each other. 

Currently having both present is flagged by as an error by the schema validation tools.

# Who is the bug affecting?

Anyone who tries to use both the rlink and base64 fields in the resource assembly.

# What is affected by this bug?

The OSCAL file author does not have an intended feature/option.

# When does this occur?

This has been observed working with the published XMD file for the XML-based OSCAL SSP model.

# How do we replicate the issue?

Put this code at the end of the back-matter:
```
      <resource id="att-diagram-network-1">
         <desc>The primary network diagram.</desc>
         <rlink href="./diagram.jpg" media-type="image/jpg"/>
         <base64>00000000</base64>
      </resource>

```

# Expected behavior (i.e. solution)

When using the above XML content in the back-matter section of an OSCAL document, the above content should be valid when checked with a published OSCAL schema validation file.

# Other Comments

- This was only verified in the SSP model; however, back-matter syntax should be identical in all models, thus all models should be verified.

- @wendellpiez investigated this, and believes it is a modeling issue in the metaschema file itself, thus is likely to be effecting all formats and OSCAL models. The NIST OSCAL team needs to decided if this is intended behavior or if the metaschema should be updated.

**Comment by wendellpiez** [comment]
Not seeing a compelling reason why the choice is exclusive, I changed the model in the metadata metaschema to permit both `rlink` and `base64` inside `resource`. PR is #516.

**Comment by david-waltermire-nist** [comment]
This was addressed in PR #516.
